File: deja-vu/DejaVu.c
#include <stdio.h>
#include <string.h>

int main() {
  char string[200];
  fgets(string, 200, stdin);
  
  int isDejaVu = 0;
  
  for (char *c = string; *c; ++c)
    if (isDejaVu = strchr(c + 1, *c)) break;
    
  if (isDejaVu) puts("Deja Vu");
  else puts("Unique");

  return 0;
}

File: deja-vu/DejaVu.java
import java.util.Scanner;
import java.util.regex.Pattern;

public class DejaVu
{
    public static void main(String[] args) {
        var input = new Scanner(System.in);
        var text = input.nextLine();
	    
        boolean isDejaVu = Pattern.matches(".*(.).*\\1.*", text);
        System.out.println(isDejaVu ? "Deja Vu" : "Unique");
   }
}
